erpnext/hr/doctype/grievance_type/grievance_type.json

```
@@ -0,0 +1,97 @@
+{
+ "actions": [],
+ "autoname": "Prompt",
+ "creation": "2021-05-11 12:41:50.256071",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "is_applicable_for_suspension",
+  "number_of_days_for_suspension",
+  "column_break_3",
+  "is_applicable_for_pay_cut",
+  "section_break_5",
+  "description"
+ ],
+ "fields": [
+  {
+   "default": "0",
+   "fieldname": "is_applicable_for_suspension",
+   "fieldtype": "Check",
+   "label": "Is Applicable for Suspension"
+  },
+  {
+   "depends_on": "eval: doc.is_applicable_for_suspension == 1",
+   "fieldname": "number_of_days_for_suspension",
+   "fieldtype": "Int",
+   "label": "Number of days for Suspension",
+   "mandatory_depends_on": "eval: doc.is_applicable_for_suspension == 1"
+  },
+  {
+   "fieldname": "column_break_3",
+   "fieldtype": "Column Break"
+  },
+  {
+   "default": "0",
+   "fieldname": "is_applicable_for_pay_cut",
+   "fieldtype": "Check",
+   "label": "Is Applicable for Pay-cut "
```

```suggestion
   "label": "Is Applicable for Pay Cut "
```
We can also shift this checkbox to the left instead of adding a column break since it is only 1 field.

---

Done